kafka/configuration.go

```diff
@@ -29,7 +29,17 @@ import (
 	"github.com/rs/zerolog/log"
 )
 
-// BrokerConfiguration represents configuration of Kafka broker
+// SASLConfiguration represents configuration of SASL authentication for
+// a given Kafka broker
+type SASLConfiguration struct {
+	SecurityProtocol string `mapstructure:"security_protocol" toml:"security_protocol"`
+	CertPath         string `mapstructure:"cert_path" toml:"cert_path"`
+	SaslMechanism    string `mapstructure:"sasl_mechanism" toml:"sasl_mechanism"`
+	SaslUsername     string `mapstructure:"sasl_username" toml:"sasl_username"`
+	SaslPassword     string `mapstructure:"sasl_password" toml:"sasl_password"`
+}
+
+// BrokerConfiguration represents configuration of a single-instance Kafka broker
 type BrokerConfiguration struct {
 	Address          string        `mapstructure:"address" toml:"address"`
 	SecurityProtocol string        `mapstructure:"security_protocol" toml:"security_protocol"`
```

Comment:
Is this line redundant with the previous `SASLConfiguration` struct field `SecurityProtocol`?

Reply:
the `SecurityProtocol` has been moved from `SASLConfiguration` to the `BrokerConfiguration` itself in the clowder library. I adapted our `BrokerConfiguration` struct to that